examples/tests/sdk_tests/queue/push.test.w

```diff
@@ -13,6 +13,14 @@ new std.Test(inflight () => {
     }
   };
 
+  let var error = false;
+  try {
+    q.push("Foo", "");
+  } catch e {
+    error = true;
```

Comment:
Can you please if the error message thrown is "Empty messages are not allowed"?
Can you please add another test case for `q.push("");`? (a single argument)

Reply:
Added in https://github.com/winglang/wing/pull/4301/commits/955712415f55a6c9f1501869de82575ea35104d9